Documentation/General/BuildSystem/HowToBuildAndroid.md

```diff
@@ -0,0 +1,54 @@
+How to build for Android {#HowToBuildAndroid}
```

Comment:
Add a page to the Wiki:
https://github.com/ezEngine/ezEngine/wiki/Building-ezEngine